app/views/planning_applications/validation/time_extension_validation_requests/_show.html.erb

```diff
@@ -29,10 +29,18 @@
       <% if @validation_request.open? %>
         Sent on <%= @validation_request.created_at.to_date.to_fs %>. Agent or applicant has not yet responded.
       <% else %>
+        <strong>Status:</strong><br>
         <%= @validation_request.approved? ? 'Approved' : 'Rejected' %>
       <% end %>
     </p>
 
+    <p class="govuk-body">
+      <% if @validation_request.rejected? && @validation_request.reason.present? %>
```

Comment:
This should always be present if it's been rejected? I think we have validation on it?